.github/workflows/build.yml

```diff
@@ -0,0 +1,81 @@
+name: build
+on:
```

Comment:
I'm not sure what kind of triggers we want to keep. Especially I'm not convinced if we need `on push` on. WDYT?

Reply:
For us `on push` only runs after PR merge. Indeed it's kind of already tested by the `pull_request` trigger (during PR).
*But*, the post-merge code can differ from PR code, because GitHub can rebase for us. I'd leave it in.